.release-it.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  git: {
+    // eslint-disable-next-line no-template-curly-in-string
+    commitMessage: 'release: v${version}',
+    // eslint-disable-next-line no-template-curly-in-string
+    tagName: 'v${version}',
+  },
+  github: {
+    release: true,
+  },
+  hooks: {
+    'after:bump': 'npm run build',
+  },
+  plugins: {
+    '@release-it/conventional-changelog': {
+      preset: 'conventionalcommits',
+      types: [
```

Comment:
Hmm I'm not sure it's using the types config – investigating...

Reply:
Fixed this – updated changelog here: https://github.com/defencedigital/moduk-frontend/actions/runs/3542455444/jobs/5948025018